tests/test_heroku.py

```diff
@@ -766,3 +784,38 @@ def test_as_context_manager(self, config, env, output, clear_workers):
         with HerokuLocalWrapper(config, output, env=env) as heroku:
             assert heroku.is_running
         assert not heroku.is_running
+
+
+class TestHerokuInfo(object):
+
+    @pytest.fixture
+    def info(self):
+
+        from dallinger.heroku.tools import HerokuInfo
+        yield HerokuInfo(team="fake team")
+
+    def test_login_name(self, info, custom_app_output):
+        login_name = info.login_name()
+        custom_app_output.assert_has_calls([
+            mock.call(['heroku', 'auth:whoami'])
+        ])
+        assert login_name == 'test@example.com'
+
+    def test_all_apps(self, info, custom_app_output):
+        app_info = info.all_apps()
+        custom_app_output.assert_has_calls([
+            mock.call(['heroku', 'apps', '--json', '--org', 'fake team'])
+        ])
+        assert len(app_info) == 2
```

Comment:
I think it would be worth comparing to the literal output value, for documentation. (Or if that's not your style, maybe put it in `all_apps.__doc__`). 

Same goes for the output of `my_apps()` below.

Reply:
In case it saves you any time, `all_apps()`:
```
[{'created_at': '2018-01-01T12:00Z',
  'name': 'dlgr-my-uid',
  'web_url': 'https://dlgr-my-uid.herokuapp.com'},
 {'created_at': '2018-01-02T00:00Z',
  'name': 'dlgr-another-uid',
  'web_url': 'https://dlgr-another-uid.herokuapp.com'}]
```

`my_apps()`:
```
[{'created_at': '2018-01-01T12:00Z',
  'dallinger_uid': 'my-uid',
  'name': 'dlgr-my-uid',
  'web_url': 'https://dlgr-my-uid.herokuapp.com'}]
```